src/bp/admin/ui/src/workspace/bots/reducer.ts

```diff
@@ -24,6 +26,8 @@ interface BotState {
   workspace?: { name: string; pipeline: any }
   // Sets the current bot used by workspace apps
   workspaceAppsBotId?: string
+  // Fetch the NLULanguages
+  botNLULanguages: string[]
```

Comment:
This property is already scoped under a `bots` key so I don't think you need to prefix its name with "bot". `nluLanguages`, `availableLanguages` or simply `languages` is enough IMHO.

Reply:
I will go with `nluLanguages`. `languages` is already used inside the bot variable. It can be confusing

If we have something like this 

```
if (bot.languages.length === languages.length)
```

```
if (bot.languges.length === nluLanguages.length)
```

I found it easier to read.